### DISCUSSION TITLE
All programs in two sections

### DISCUSSION THREAD

**Discussion by perdrix52** [Open-Shell/Open-Shell-Menu#968] [Q&A] [answered]
When I click on All Programs the list of folders and applications is opened with quite a few programs at the top, then a set of folders then the Apps and then another set of Folders 

![image](https://user-images.githubusercontent.com/20211762/153766169-010be8a5-9984-4b95-8ef8-4b80f926bf65.png)

I expected the second set of folders to be sorted in with the others - why aren't they?

D.

**Comment by Ibuprophen** [comment, marked as answer]
Duplicate of https://github.com/Open-Shell/Open-Shell-Menu/issues/969. 


~Ibuprophen
